src/Tests/Singularity.Test/Extensions/ModuleExtensionsTests.cs

```diff
@@ -13,5 +13,16 @@ public void ToBindings_SingleModule()
 			var instance = container.GetInstance<ITestService10>();
 			Assert.IsType<TestService10>(instance);
 		}
+
+        [Fact]
+        public void RegisterModule_Generic()
+        {
+			var container = new Container(cb =>
```

Comment:
Seems tabs and spaces are getting mixed here? Please use spaces.

Reply:
I, again, apologize for this. I thought I had set my IDE to use spaces instead of tabs. In addition, I just submitted a change to address this specifically where I re-indented the entire method, but it appears to have made things worse? I think I may have the wrong tab size set for this. I currently have 4 for tab size and 4 for indent size. What should I set these to so that they are consistent with your coding style?

![image](https://user-images.githubusercontent.com/632589/92271375-ec9d4b00-eea4-11ea-9edb-2b1c5b628465.png)

Reply:
Hmm in the code of the PR it still using tabs? Maybe just force a reformat so that the new settings get applied.

Having a editorconfig would help to keep things consistent yeah, haven't really spent effort on that since iam pretty much the only one working on this so far.

Reply:
I submitted an `.editorconfig` with some of the settings that I used in other projects before. By no means am I implying that the code should be styled according to the settings I submitted. Please make the changes you see fit to adhere to your coding style.

Reply:
Made some changes to the editorconfig you added in https://github.com/Barsonax/Singularity/pull/147. I think its better to start with a minimal editorconfig and let it grow over time.

Closing this PR as #147 is a continuation of this one.